test/test_helper.rb

```diff
@@ -30,6 +30,14 @@
 
 module ActiveSupport
   class TestCase
+    setup do
+      Timecop.freeze(Time.zone.local(2022, 11, 7))
```

Comment:
I'm sorry to be that person, but could we pick a different date? Any other date is fine, just this one has complicated significance for me.

Reply:
of course